server/athenian/api/controllers/miners/github/contributors.py

```diff
@@ -81,17 +94,25 @@ async def fetch_releases():
     data = await asyncio.gather(
         fetch_prs(), fetch_comments(), fetch_commit_authors(), fetch_commit_committers(),
         fetch_reviews(), fetch_releases())
-    users = {}
+
+    stats = defaultdict(dict)
```

Comment:
`marshal` does not support `defaultdict`, so I used a regular dict here. We don't expose it outside now so OK.

Reply:
thanks for pointing it out 👍